spec/models/course_year_spec.rb

```diff
@@ -23,4 +23,26 @@
     it { is_expected.to validate_presence_of(:title).with_message("Enter a title") }
     it { is_expected.to validate_presence_of(:content).with_message("Enter content") }
   end
+
+  describe "course_modules" do
+    it "returns modules in order after they get updated" do
+      course_year = FactoryBot.create(:course_year)
+      course_module_one = FactoryBot.create(:course_module, title: "One", course_year: course_year)
+      course_module_two = FactoryBot.create(:course_module, title: "Two", course_year: course_year)
+      course_module_three = FactoryBot.create(:course_module, title: "Three", course_year: course_year)
+      course_module_four = FactoryBot.create(:course_module, title: "Four", course_year: course_year)
+
+      course_module_two.previous_module = course_module_one
```

Comment:
Just wondering if course_module_two.update!(previous_module: course_module_one) wouldn't be better